configs/permissions/insights.json

```diff
@@ -1,3 +1,7 @@
 {
-    "*": ["*"]
+    "*": ["*"],
+    "acks": ["read", "write", "*"],
+    "preferences": ["read", "write", "*"],
```

Comment:
@dkuc let's remove this altogether - on the user preferences page, we allow users to set their own preferences for any application they have regular RBAC permissions to; no need for an extra preferences permission in any app.

Reply:
How will the advisor api know to accept a call to update preferences?

Reply:
@dkuc how does the advisor API know when user preferences are updated for Advisor today? currently the user preferences page is checking for the user to have app permissions, not user preferences permissions. I'm not sure exactly how the interactions work app-side but I would assume that since this works today, it would continue to work that way and is not impacted by custom role app permissions. I may be missing some consideration so I'm happy to chat about this on a call if that's easier.